packages/wallet-ui/src/services/useStarkNetSnap.ts

```diff
@@ -191,6 +192,52 @@ export const useStarkNetSnap = () => {
     dispatch(disableLoading());
   };
 
+  const hideAccount = async ({
+    chainId,
+    address,
+    currentAddress,
+  }: {
+    chainId: string;
+    address: string;
+    currentAddress: string;
+  }) => {
+    try {
+      if (!loader.isLoading) {
+        dispatch(
+          enableLoadingWithMessage(`Hiding account ${shortenAddress(address)}`),
+        );
+      }
+      const account = await toggleAccountVisibility(chainId, address, false);
+      dispatch(updateAccount({ address, updates: { visibility: false } }));
+      if (account.address !== currentAddress) {
+        await initWalletData({
+          account,
+          chainId,
+        });
+      }
+    } catch (error) {
+      //eslint-disable-next-line no-console
+      console.log(`error while processing hideAccount: ${error}`);
```

Comment:
he invokeSnap will log for you, instead may be throw an error and let the consumer to have a toast message to tell the user hiding is fail? or you can directly toast from here

```suggestion
      console.log(`error while processing hideAccount: ${error}`);
```

Reply:
let's directly toast from here!